docs/SUMMARY.md

```diff
@@ -35,7 +35,9 @@
         * [SQL Databases](connectors/sql.md)
         * [Turbopuffer](connectors/turbopuffer.md)
         * [Unity Catalog (Databricks)](connectors/unity_catalog.md)
-    * [Scaling Out and Deployment](distributed.md)
+    * Scaling Out and Deployment
+        * [Distributed Computing](distributed.md)
```

Comment:
Should we be explicit now and call out that this is how to run Daft on Ray?

I.e. presenting running daft distributed as: you can choose to run us on Ray, OR run us on Kubernetes.

Reply:
This makes sense to me too